spec/models/workshop_invitation_spec.rb

```diff
@@ -1,37 +1,70 @@
 require 'spec_helper'
 
 RSpec.describe WorkshopInvitation, type: :model  do
-  context 'methods' do
-    let(:invitation) { Fabricate(:student_workshop_invitation) }
-    let!(:accepted_invitation) { 2.times { Fabricate(:workshop_invitation, attending: true) } }
+  it_behaves_like InvitationConcerns, :workshop_invitation
 
-    it_behaves_like InvitationConcerns
+  context 'defaults' do
+    it { is_expected.to have_attributes(attending: nil) }
+    it { is_expected.to have_attributes(attended: nil) }
+  end
+
+  context 'validates' do
+    it { is_expected.to validate_presence_of(:workshop) }
+    it { is_expected.to validate_uniqueness_of(:member_id).scoped_to(:workshop_id, :role) }
+    it { is_expected.to validate_inclusion_of(:role).in_array(['Student', 'Coach']) }
   end
 
   context 'scopes' do
-    it '#attended' do
-      4.times { Fabricate(:attended_workshop_invitation) }
+    context '.attended' do
```

Comment:
can we stick to the `#method_tested` notation that is already used across specs please.